browser/.env

```diff
@@ -1,2 +1,3 @@
 REACT_APP_TITLE=Graph Browser
-REACT_APP_REPO_URL=https://github.sc-corp.net/Snapchat/android
\ No newline at end of file
+REACT_APP_REPO_URL=https://github.sc-corp.net/Snapchat/android
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
what's this for?

Reply:
My issue was related to this except the dependency was an older version of Jest.
https://stackoverflow.com/questions/56528222/npm-start-returns-error-there-might-be-a-problem-with-the-project-dependency-t

mine said something like this 
The react-scripts package provided by Create React App requires a dependency:

"jest": "26.6.0"

** I'm on jest ^27.x.x

line 3 basically bypasses that check and allows the project to build. I couldn't find anything that could cause future issues for the dagger browser.